packages/web-app-files/src/components/SideBar/Shares/Collaborators/ListItem.vue

```diff
@@ -312,6 +312,12 @@ export default defineComponent({
         title: this.$gettext('Access details'),
         list
       }
+    },
+    showAvatar() {
+      if (this.isSpace) {
+        return !!this.share.collaborator.name
```

Comment:
Is there really no other prop available? This looks too hacky 😢

Reply:
Unfortunately not :( As stated above, space user- and group-shares have the exact same share type at the moment. I don't know the reasoning behind that, but that's how the server handles it. Hence Web does the same.

You could say it is somewhat logical because the collaborator name is used to fetch the user avatar (and falls back to initials). So if no name given -> no avatar. Not super convincing, I know...

Reply:
Had another look... the prop is only empty on existing group-type members. If you add a group the `share.collaborator.name` prop holds the group name, thus still no group icon visible. :-(

Reply:
Damn it... We need to think of a solution here. Also, I found more bugs with space group shares, which are probably related to the missing name as well. I will create issues for it.

Reply:
Also see my comment in https://github.com/owncloud/web/pull/8300#discussion_r1084893140.

Reply:
Resolved by introducing a `shareType 8` in web. The backend also understands the shareType now when creating or updating a space membership. For listing space memberships the shareType was only ever client side anyway. Considering this as resolved. :-)